usagov/usagov-2021
# Add benefit-finder integration test github workflow

## Jira Task
USAGOV-1909
https://cm-jira.usa.gov/browse/USAGOV-1901


## Description
This PR adds a github workflow file in .github/workflows/benefits-finder-test.yml

## Type of Changes

  - Add screenshot showing what it should look like
- [x ] Infrastructure

## Testing Instructions
This pipeline should pull in the benefits-finder latest release code and merge it with the branch it is run against (in this case dev). Then execute some browser testing against it to try and catch issues earlier.

### Change Requirements


### Validation Steps

- Execute pipeline at will as it is run on a dispatch basis. 
- Ensure the testing pipeline runs correctly and does not fail to complete all of its browser testing.
- Review any errors it finds to determine their accuracy and solutions to an found issues.

## Security Review




## Reviewer Reminders

- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions

Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
2. Find the commit of this pull request.
3. Build and deploy the changes.
4. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
